unity-client/Assets/Scripts/MainScripts/DCL/Components/UI/UIImage/UIImage.cs

```diff
@@ -59,7 +59,7 @@ public override IEnumerator ApplyChanges(string newJson)
                 {
                     if (fetchRoutine != null)
                     {
-                        scene.StopCoroutine(fetchRoutine);
+                        CoroutineStarter.Stop(fetchRoutine);
```

Comment:
Shouldn't we also Stop coroutine when Dispose is called?

Reply:
good catch, applied!